src/crd-controller/crd-controller.csproj

```diff
@@ -0,0 +1,36 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <ServerGarbageCollection>false</ServerGarbageCollection>
+    <RootNamespace>CustomResourceDescriptorController</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Folder Include="wwwroot\" />
+    <Folder Include="HostedServices\" />
+    <Folder Include="Models\" />
+    <Folder Include="HealthChecks\" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="KubernetesClient" Version="1.5.1-g60ffd20a7c" />
+    <PackageReference Include="System.Net.Http" Version="4.3.3" />
+    <PackageReference Include="App.Metrics.AspNetCore.Mvc" Version="2.0.0" />
+    <PackageReference Include="App.Metrics.Formatters.Prometheus" Version="2.0.0" />

```

Comment:
Please update both AppMetrics packages to version 3 for newest Prometheus support

Reply:
I will!